CMakeLists.txt

```diff
@@ -20,8 +34,11 @@ target_include_directories(meshio INTERFACE
 
 target_compile_features(meshio INTERFACE cxx_std_17)
 
-include(CTest)
-add_subdirectory(test)
+if((CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME OR MODERN_CMAKE_BUILD_TESTING)
```

Comment:
MODERN_CMAKE_BUILD_TESTING what is this variable. I can't find in regular CMake docs ?

Reply:
My mistake, I blindly copied from other project without looking at what I was typing. Removed this now.